package/score-worker/src/main/java/io/cloudslang/schema/ConfValue.java

```diff
@@ -50,6 +50,10 @@ public void register(Element element, ParserContext parserContext){
 	}
 
 	private Object fromString(String value) {
+		String sysValue = System.getProperty("cloudslang.worker." + name);
```

Comment:
It was intended for worker properties only. It’s hard to change it now since it should be backwards  compatible from 10.22 to 10.50 and in 10.22 it’s cloudslang.worker.
From: Meir Wahnon [mailto:notifications@github.com]
Sent: יום א 05 יולי 2015 13:03
To: CloudSlang/score
Cc: Eskin, Alexander
Subject: Re: [score] Load config value from Java system properties first, then from the xml (#50)


In package/score-worker/src/main/java/io/cloudslang/schema/ConfValue.java<https://github.com/CloudSlang/score/pull/50#discussion_r33893502>:

> @@ -50,6 +50,10 @@ public void register(Element element, ParserContext parserContext){

>       }

>

>       private Object fromString(String value) {

> +            String sysValue = System.getProperty("cloudslang.worker." + name);

"cloudslang.worker." , why worker? ConfValue is not only for the worker part...
maybe "cloudslang.conf." or "cloudslang.env."...

—
Reply to this email directly or view it on GitHub<https://github.com/CloudSlang/score/pull/50/files#r33893502>.